### PR TITLE
Project refactor

### PR DESCRIPTION
**Pull request by alanluqman**
## File Changed ♻️ 
- style.css
- index.html
- script.js

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+const library = [];
+// class book {
+//   constructor(title, author) {
+//     this.title = title;
+//     this.author = author;
+//   }
+// }
```
Comment (TedLivist) [inline app.js:7]: - [ ] Kindly remove commented code here. It makes your codebase look messy and can be confusing.

File: app.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  list.display();
+}
+
+const add = document.getElementById('add_btn');
+add.addEventListener('click', () => {
+  const title = document.getElementById('title').value;
+  const author = document.getElementById('author').value;
+  list.addBook(title, author);
+  list.display();
+  // console.log(library);
```
Comment (TedLivist) [inline app.js:71]: - [ ] Kindly remove commented code here.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @alanluqman and @Berkay-akbas,

Good job so far!
✔️ Use of classes
✔️ Books can be added and removed
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] Kindly format your entire README to follow the [Microverse template](https://github.com/microverseinc/readme-template)

  bag

- [x] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions:
- [ ] Kindly add some instructions in your README file on how to clone the project and run it locally to make your README more professional.
- [ ] Please clear the inputs of `Title` and `Author` after adding a book for a better user experience.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**